pkg/addons/default/generate.go

```diff
@@ -1,6 +1,6 @@
 package defaultaddons
 
-//go:generate curl --silent --location https://github.com/aws/amazon-vpc-cni-k8s/blob/957f835f437fd76953f37e6b409ba6426bb4ce37/config/v1.4/aws-k8s-cni.yaml?raw=1 --output assets/aws-node.yaml
+//go:generate curl --silent --location https://github.com/aws/amazon-vpc-cni-k8s/blob/723dd5a2d0387477f6950b23eba6e8b586a59a70/config/v1.5/aws-k8s-cni.yaml?raw=1 --output assets/aws-node.yaml
 //go:generate curl --silent --location https://github.com/aws/amazon-vpc-cni-k8s/blob/957f835f437fd76953f37e6b409ba6426bb4ce37/config/v1.4/aws-k8s-cni-1.10.yaml?raw=1 --output assets/aws-node-1.10.yaml
```

Comment:
Is there a new one for 1.10 also?

Reply:
No, I didn't see it in the repo. I wasn't sure if I should use the same file or keep 1.4 :/ What do you think?

Reply:
Ah, so perhaps they just decided to stop supporting 1.10, which is something that's meant to happen sooner or later in general.